versioned/spi/src/main/java/org/projectnessie/versioned/VersionStore.java

```diff
@@ -123,13 +125,17 @@ Hash commit(
    * @param referenceHash The hash to use as a reference for conflict detection. If not present, do
    *     not perform conflict detection
    * @param sequenceToTransplant The sequence of hashes to transplant.
+   * @param updateCommitMetadata Function to allow resetting properties application due to merge.
```

Comment:
Please reword. "resetting properties application" is confusing. `merge` is wrong.

Reply:
ok